src/lib.rs

```diff
@@ -26,6 +26,7 @@
         target_os = "emscripten"
     ))
 ))]
+#[cfg(feature = "wayland")]
```

Comment:
I feel like a lot of this should be simplified by just updating `copypasta` to Rust 2018, which removes all annotations on the `extern crate` imports. I've created a PR for that [here](https://github.com/alacritty/copypasta/pull/8).

That way we can just exclude the modules and `ClipboardContext` and we should be good to go with 3 code annotations total (I think).